Cloud changelog - Oct 6th (#5905)
mattermost/docs: source/install/cloud-changelog.md

Latest Mattermost Cloud releases:

- [Release 2022-10-06](#release-2022-10-06)
- [Release 2022-09-15](#release-2022-09-15)
- [Release 2022-09-08](#release-2022-09-08)
- [Release 2022-09-01](#release-2022-09-01)
- [Release 2022-08-25](#release-2022-08-25)
- [Release 2022-08-18](#release-2022-08-18)

## Release 2022-10-06

### Improvements

#### User Interface (UI)
 - Added Insights to the channel switcher.
 - Added a button to easily copy the content of text or code files in file previews.
 - The team unread icon for muted channels is now hidden in the sidebar.
 - Updated the designs for the “payment failed” email notifications.

#### Administration

>  - A ``batchSize`` option has been added to the ``mattermost export`` CLI command to limit the number of items exported. By default, if it is not included, it exports all posts.

Reviewer: do we mean `mmctl` export?

Reviewer 2: If we are referring to the CLI command, docs updates for v7.4 will be available here: https://docs.mattermost.com/manage/command-line-tools.html#mattermost-export

Author: This is referring to the CLI command.

Latest Mattermost Cloud releases:

- [Release 2022-10-06](#release-2022-10-06)
- [Release 2022-09-15](#release-2022-09-15)
- [Release 2022-09-08](#release-2022-09-08)
- [Release 2022-09-01](#release-2022-09-01)
- [Release 2022-08-25](#release-2022-08-25)
- [Release 2022-08-18](#release-2022-08-18)

## Release 2022-10-06

### Highlights

#### Message Priority
 - Added an option to select a message priority label for root posts. Labels are ``standard``, ``important``, and ``urgent``.

### Improvements

#### User Interface (UI)
 - Added Insights to the channel switcher.
 - Added a button to easily copy the content of text or code files in file previews.
 - The team unread icon for muted channels is now hidden in the sidebar.
 - Updated the designs for the “payment failed” email notifications.

#### Administration
 - A ``batchSize`` option has been added to the mattermost export CLI command to limit the number of items exported. By default, if it is not included, it exports all the posts.

>  - Added a credit card update page that communicates arrears for Admins.

Author: Adding @nickmisasi to help clarify.

#### API Changes
 - Updated the Files search and get APIs to filter out files, past the set Cloud plan's limit.
 - Added a new response-header ``First-Inaccessible-File-Time`` to the APIs fetching single file info.
 - Added a new query parameter to include deleted posts as long as it's requested by a System Admin in ``/api/v4/channels/{channel_id}/posts``.

### Bug Fixes
 - Fixed an issue with the badge count on the mobile app when a channel/thread was viewed.
 - Fixed an issue where typing ``@`` in the right-hand side rendered a cut-off user suggestion list.
 - Fixed an issue where an error screen was briefly flashed when the first Admin signed up into a new server.
 - Fixed an issue where users were unable to add Japanese comments correctly in the message **Forward** modal.
 - Fixed an issue where unsaved edits to a post were lost when switching channels or threads.
 - Fixed an issue on larger screen sizes where the Insights widgets were pushed to the side when the right-hand side was open.

>  - Fixed an issue where the ability to forward messages from public channels to someone whom you've never Direct Messaged before wasn’t possible.

Reviewer 2: ```suggestion
 - Fixed an issue where the ability to forward messages from public channels wasn't possible when messaging someone directly for the first time.
```

 - Fixed an issue where custom emojis were sometimes not visible in **Insights > Top Reactions**.

>  - Fixed an issue where channels with no posts in a particular time window didn't show in **Insights > Least Active Channel**.

Reviewer 2: ```suggestion
 - Fixed an issue where channels with no posts within a particular timeframe didn't show in **Insights > Least Active Channel**.
```
